batchflow/models/torch/layers/core.py

```diff
@@ -49,24 +49,59 @@ class Dense(nn.Module):
     keep_dims : int, optional
         Dimensions to keep while flattening input, see :class:`~.Flatten`, by default 0.
     """
-    def __init__(self, units, bias=True, inputs=None, flatten=True, keep_dims=0):
+    def __init__(self, features, bias=True, inputs=None, flatten=True, keep_dims=0):
```

Comment:
If we try namings like in [torch](https://pytorch.org/docs/stable/generated/torch.sum.html) why not `keepdim` instead of `keep_dims`?